### ISSUE TITLE
User interface improvements: Calculations page

### ISSUE THREAD

**Issue by entrotech** [hackforla/tdm-calculator#108]

**Comment by nicholaspung** [comment]
Line 17 of [rosemary requirements](https://drive.google.com/open?id=1F03uD-yk9aPuKoc6YuRyCI_kvZRCORwn)

**Comment by ExperimentsInHonesty** [comment]
snipits from slack
@entrotech 12/31/19
Putting the Next/Previous page buttons at the bottom of the screen makes it a bit clumsy to navigate on the pages where the content pushes the button off the bottom of the screen, since you need to scroll down to access them.  I'm still fighting with vertical sizing/scrolling  implementation, but want the left side panel to stay put, and the right panel to scroll vertically when the content extends beyond the page.

@nclairesays   
... I checked out the site. In terms of the next/prev page buttons being at the bottom, you can set the “justify-content” to “flex-start” or just remove it all together and it looks good and pushes the arrows closer to the content
![image (3)](https://user-images.githubusercontent.com/37763229/71698715-5347f300-2d71-11ea-8283-37ce073b134d.png)

@entrotech 
The problem is that the content can require a lot of height.  If you check all the land use check boxes, then go to the next page (Project Parameters), you will see that you need to scroll way down to even get to the end of the content.

@nclairesays  
maybe we can put the arrows at the top of the page and the bottom? or on the left side bar?
i think placing it top and bottom will solve that problem. It could be a little smaller/subtler so arrows / bold yellow boxes are  not just in your face 

![image (4)](https://user-images.githubusercontent.com/37763229/71698780-95713480-2d71-11ea-8a55-bd32e230432a.png)
![image (5)](https://user-images.githubusercontent.com/37763229/71698790-9f933300-2d71-11ea-9f37-d772f1115ab4.png)

maybe? (at work/lunch break right now so can’t play with the actual code.)
changes -->
button padding: .5em 1em;
removed margin-bottom on the div containing buttons
added margin-top to the content div 1em

**Comment by ExperimentsInHonesty** [comment]
snipits from slack and phone
@ExperimentsInHonesty 01/01/20
Please remove or reduce the distance on the drop shadow, right now it just looks like a dirty blur

@ExperimentsInHonesty  - directly to @entrotech via both phone and slack
The blur is less far away in @nekobox's figma - see images.  Also the drop shadow should be on the button.

Figma (design spec)
![Screen Shot 2020-01-02 at 3 14 34 PM](https://user-images.githubusercontent.com/37763229/71699047-aec6b080-2d72-11ea-8f32-949e2a65a7be.png)

Heroku (Live app)
![Screen Shot 2020-01-02 at 3 16 02 PM](https://user-images.githubusercontent.com/37763229/71699087-dc135e80-2d72-11ea-9ee2-aee0455dd634.png)

Figma (design spec)
![Screen Shot 2020-01-02 at 1 44 41 PM](https://user-images.githubusercontent.com/37763229/71699151-2268bd80-2d73-11ea-8545-49c8264bb228.png)

Heroku (Live app)
![Screen Shot 2020-01-02 at 3 18 54 PM](https://user-images.githubusercontent.com/37763229/71699182-3ca29b80-2d73-11ea-9d71-125636cfdfa8.png)


So to summarize:
The drop shadow should be closer to the image it is a shadow of.  
If the image is on a button, there is no drop shadow on the image, only on the button itself.

**Comment by entrotech** [comment]
PR #134 and PR #135 should bring the TDM Calculation Wizard much closer to the wireframe layout. It's not pixel-perfect, but now that it's deployed, we call all play with it and do the fine-tuning of the Calculation Wizard. Also be sure to re-size your browser window - it is not designed to work on mobile devices, but should handle a range of browser window sizes and aspect ratios on desktop.

**Comment by entrotech** [comment]
This issue is intended to only address the layout of the Calculation Screen. I think changes to the other screeens, like Login/Registration, etc. should be separate issues.

**Comment by entrotech** [comment]
I'm closing this issue as a complete first pass, per a number of recent PRs. Additional changes should be requested by creating additional, more specific request issues.
